### PR TITLE
Refine readme template

### PR DESCRIPTION
**Pull request by MingjiaLiu1995**
Refine readme template

### CODE REVIEW

File: README_TEMPLATE.md [hunk 1]
```diff
@@ -12,36 +12,50 @@
...
+
+3. **Preview Your Office Add-in (F5)**
+    
+    Select `Preview Your Office Add-in(F5)` on tree view and select a launch config. A Word/Excel/PowerPoint app will launch with add-in sample side-loaded. 
+    * To debug on Office on the web, go to [Sideload Office Add-ins to Office on the web](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing)
     * To debug in Desktop (Edge Legacy), go to [Debug Edge Legacy Webview](https://learn.microsoft.com/office/dev/add-ins/testing/debug-add-ins-using-devtools-edge-legacy)
-    * **For Office on the web**: [Sideload Office Add-ins to Office on the web](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing)
-4. Click `Stop Previewing Your Office Add-in` to stop debugging.
+    
+    **If you meet sideloading errors, please first confirm the following items. If you still have problems, please [Create an issue]( https://github.com/OfficeDev/Office-Samples/issues/new) and we'll help you out.**
```
Comment (hermanwenhe) [inline README_TEMPLATE.md:40]: Do we need to add the link of trouble shooting page of our document? like this page https://learn.microsoft.com/en-us/office/dev/add-ins/testing/troubleshoot-development-errors and clear cache page(which has a lot of traffic):  https://learn.microsoft.com/en-us/office/dev/add-ins/testing/clear-cache

Comment (hermanwenhe) [inline README_TEMPLATE.md:40]: For create issue link, should they go to office-js repo just like the issue link in toolkit?

File: README_TEMPLATE.md [hunk 2]
```diff
@@ -68,9 +82,24 @@ DELETE_THIS_LINE:Use copilot chat @workspace to generate folder structure
...
+<br><img src="./assets/github_copilot_extension.png" width="400"/>
+
+**Resources to learn more Office add-ins capabilities:**
+* Select '`View Samples`' on `Office Add-in Dev Kit` tree view for real-world examples and code structures.
+* [Read the documentation](https://learn.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins) of Office add-ins.
+
+## Engage with the team
 Did you experience any problems with the sample? [Create an issue]( https://github.com/OfficeDev/Office-Samples/issues/new) and we'll help you out.
 
+Want to learn more about new features, development practices, and additional information? [Join the Microsoft Office Add-ins community call.](https://learn.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins-community-call)
```
Comment (hermanwenhe) [inline README_TEMPLATE.md:101]: All links should remove the '\en-us\' so they can go to the localized version of the document.

File: README_TEMPLATE.md [hunk 3]
```diff
@@ -68,9 +82,24 @@ DELETE_THIS_LINE:Use copilot chat @workspace to generate folder structure
 | webpack.config.js             Webpack config
 ```
 
-## Feedback
+### Make code changes
+git push
```
Comment (hermanwenhe) [inline README_TEMPLATE.md:86]: Why is git push here.

File: README_TEMPLATE.md [hunk 4]
```diff
@@ -68,9 +82,24 @@ DELETE_THIS_LINE:Use copilot chat @workspace to generate folder structure
...
+**GitHub Copilot extension for Office Add-ins:** 
+DELETE_THIS_LINE: Only shows in projects generated by GitHub Copilot extension.
+1. Type in `@office` to invoke the extension.
+1. Type in `/generatecode` and describe the feature you would like to build, then send the request to Copilot.
```
Comment (hermanwenhe) [inline README_TEMPLATE.md:90]: Should discuss with @GavinGu07 about how we can keep this section in agent generated projects while not keeping this in toolkit generated project.

**Review by hermanwenhe — CHANGES_REQUESTED** [review]

File: README_TEMPLATE.md (hunk 4 above)
Comment (GavinGu07) [reply README_TEMPLATE.md:90]: Is this the README which is shared by both toolkit and agent? Seems like this is the only section instructing the usage of GitHub agent.

Comment (MingjiaLiu1995) [reply README_TEMPLATE.md:90]: Yes, I think it would be better using the same template for all samples to keep alignment.

File: README_TEMPLATE.md (hunk 1 above)
Comment (MingjiaLiu1995) [reply README_TEMPLATE.md:40]: I don't think these troubleshooting pages are closely related to the common sideload errors here. Maybe it's not good resources to include

File: README_TEMPLATE.md [hunk 5]
```diff
@@ -12,36 +12,50 @@
...
-https://www.microsoft.com/en-us/microsoft-365/try?rtc=1)
+https://www.microsoft.com/microsoft-365/try?rtc=1)
 or [purchase a Microsoft 365 plan](
-https://www.microsoft.com/en-us/microsoft-365/buy/compare-all-microsoft-365-products).
+https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
 
 
-### Run and debug the add-in
-1. Open Office Add-in Debugger
-<br>![](./assets/toolkit_development.png)
```
Comment (wangyun-microsoft) [inline README_TEMPLATE.md:29]: Need to update this picture, as attached. 
image (2)

File: README_TEMPLATE.md [hunk 6]
```diff
@@ -12,36 +12,50 @@
...
-2. Click `Check and Install Dependencies`
-3. Launch and debug
-    * **For Office on Windows/macOS**, click `Preview Your Office Add-in(F5)` button on tree view and select a launch config. A Word/Excel/PowerPoint app will launch with add-in sample side-loaded. **Note:** Debugging on macOS is not supported yet.
+### Run the add-in using Office Add-in Dev Kit
+[Office Add-in Dev Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) is an end-to end developer tool for building Office add-ins. You can use this tool to easily creating, running and debugging, and managing the lifecycle of an Office add-in.
+<br>![](./assets/toolkit_development_preview.png)
```
Comment (wangyun-microsoft) [inline README_TEMPLATE.md:28]: Replace the image with attached one. 
image (2)

File: README_TEMPLATE.md [hunk 7]
```diff
@@ -12,36 +12,50 @@
...
+
+1. **Check and Install Dependencies**
+
+    Select `Check and Install Dependencies` to install npm dependencies before debugging.
```
Comment (wangyun-microsoft) [inline README_TEMPLATE.md:32]: Select `Check and Install Dependencies` to check your environment and install necessary dependencies in order to run and debug the add-in code.

File: README_TEMPLATE.md [hunk 8]
```diff
@@ -12,36 +12,50 @@
...
+
+3. **Preview Your Office Add-in (F5)**
+    
+    Select `Preview Your Office Add-in(F5)` on tree view and select a launch config. A Word/Excel/PowerPoint app will launch with add-in sample side-loaded.
```
Comment (wangyun-microsoft) [inline README_TEMPLATE.md:36]: Do we have other launch configs? I'm assuming we only have one?

File: README_TEMPLATE.md [hunk 9]
```diff
@@ -12,36 +12,50 @@
...
+- [Office Add-in Dev Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) version 0.4.0 and higher.
+- Office connected to a Microsoft 365 subscription. You might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](
+https://developer.microsoft.com/microsoft-365/dev-program), see [FAQ](
+https://learn.microsoft.com/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-) for details.
 Alternatively, you can [sign up for a 1-month free trial](
-https://www.microsoft.com/en-us/microsoft-365/try?rtc=1)
+https://www.microsoft.com/microsoft-365/try?rtc=1)
 or [purchase a Microsoft 365 plan](
-https://www.microsoft.com/en-us/microsoft-365/buy/compare-all-microsoft-365-products).
+https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
```
Comment (wangyun-microsoft) [inline README_TEMPLATE.md:23]: Do we need to make sure Word, PPT or Excel should be closed?

File: README_TEMPLATE.md [hunk 10]
```diff
@@ -12,36 +12,50 @@
 ## How to run this sample
 
 ### Prerequisites
-- [Node.js](https://nodejs.org) 16/18 (Tested on DELETE_16.14.0_THIS_USE_ANOTHER_VERSION)
-- [Office Add-in Debugger](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) version 0.4.0 and higher.
-- Office connected to a Microsoft 365 subscription (including Office on the web). If you don't already have Office, you might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](
-https://developer.microsoft.com/en-us/microsoft-365/dev-program);
-for details, see the [FAQ](
-https://learn.microsoft.com/en-us/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-).
+- [Node.js](https://nodejs.org) 16 or 18 (Tested on DELETE_16.14.0_THIS_USE_ANOTHER_VERSION) and [npm](https://www.npmjs.com/get-npm). To verify if you've already installed these tools, run the commands node -v and npm -v in your terminal.
```
Comment (wangyun-microsoft) [inline README_TEMPLATE.md:15]: The latest node.js is 20. I know some examples need specific node version like 18 or 16, but generally we should support the latest node version.

File: README_TEMPLATE.md (hunk 8 above)
Comment (wangyun-microsoft) [inline README_TEMPLATE.md:36]: Select `Preview Your Office Add-in(F5)` on the side panel to start running and debugging the add-in code. A Word/Excel/PowerPoint app will launch with the add-in sample side-loaded.

Comment (MingjiaLiu1995) [reply README_TEMPLATE.md:36]: No. we only have one for desktop, but developers still need to select

File: README_TEMPLATE.md (hunk 9 above)
Comment (MingjiaLiu1995) [reply README_TEMPLATE.md:23]: The note is in the sideload section - Line43
